### ISSUE TITLE
Subtitle editor, browser scroll-bar overlaps scroll-bar / time-line of the video

### ISSUE THREAD

**Issue by cgruber-uos** [opencast/opencast-editor#931]
To make navigation easy, especially for a long video, the CC-segment and wave graph is wider than the browser window. This allowas sophisticated navigation tasks. So it looks good at the first run: 

![2022-12-07 11_41_26-Greenshot](https://user-images.githubusercontent.com/101632236/206158295-96588ed4-c2d3-4e0f-af07-2a4d28b6b887.png)

But in case I try to drag'n'drop the **video scroll bar** a second scroll bar shows up, the **browser scroll bar** and interference with the first **video scroll bar** due to overlapping: 

![2022-12-07 11_41_27-Greenshot](https://user-images.githubusercontent.com/101632236/206158309-8edeb759-3e7a-492d-bdca-59642a9d0c09.png)

Drag'n'drop the video scroll bar to navigate in the video is nearly not possible due to a really narrow area of the scroll bar!

**Comment by Arnei** [comment]
Assuming you're referring to the grey bar at the far bottom of the page when talking about **video scroll bar** (also called "Mini Timeline" in the tooltip). 

You cannot drag n' drop on the "Mini Timeline" to navigate the video. The "Mini Timeline" only supports clicking. Drag n' drop for navigation only works on the waveform part of the timeline (and technically the empty spaces below and above the waveform).

Regarding the actual issue of the horizontal browser scroll bar overlapping the "Mini Timeline": That seems to be fixed on `main`. At least in my up-to-date Firefox and Chrome. Could you test again on https://editor.opencast.org/ and if the issue is still there for you, report the browser and browser version you are using?

**Comment by cgruber-uos** [comment]
You are right, just "clicking", drag and drop ist just working for the browser scroll bar. 

The issue is **not fixed** at https://editor.opencast.org/ [2022-12-08] with Win11 on **FF 107.0.1** (= latest) You are getting overlapping elements: 

![2022-12-08 12_42_58-Greenshot](https://user-images.githubusercontent.com/101632236/206438896-52ba34f4-fddb-4955-b584-203165acbd1c.png)

But it's working on **Chrome**, there is seperate space for both navigation bars: 

![2022-12-08 12_44_56-Greenshot](https://user-images.githubusercontent.com/101632236/206439023-a7148128-fba6-4a91-b5cd-3bff30698470.png)

... and Chromium is working and showing more space to the page end (but other error, video could not be load): 

![2022-12-08 12_46_12-Greenshot](https://user-images.githubusercontent.com/101632236/206439143-1a07dd26-d4bb-4ede-a17c-8135932f60bd.png)

**Comment by Arnei** [comment]
Fixed by #1673
